<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1430</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign up</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">/*white widget*/
#widget_2{
	border-radius: 10px;
}

QLabel{
	color: #888888;
}

QLineEdit{
	border: none;
	border-bottom: 1px solid #cccccc;
}

QLineEdit:focus{
	border: none;
	border-bottom: 1px solid rgb(27, 121, 209);
}

QPushButton#SignUp_button{
	color: rgb(27, 121, 209);
	border: 1px solid rgb(27, 121, 209);
	border-radius: 5px;
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="SignUp">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QWidget" name="CenterWidget" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>700</width>
          <height>500</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QWidget" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>280</height>
            </size>
           </property>
           <property name="font">
            <font>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
                <property name="spacing">
                 <number>6</number>
                </property>
                <item>
                 <widget class="QLabel" name="Welcome">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <family>DejaVu Serif</family>
                    <pointsize>15</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(27, 121, 209);</string>
                  </property>
                  <property name="text">
                   <string>               Welcome</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Username_label">
                  <property name="font">
                   <font>
                    <family>DejaVu Serif</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Username</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Username_input">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Password_label">
                  <property name="font">
                   <font>
                    <family>DejaVu Serif</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Password_input">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Password2_label">
                  <property name="font">
                   <font>
                    <family>DejaVu Serif</family>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Repeat password</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::RichText</enum>
                  </property>
                  <property name="margin">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Password2_input">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="Password_mismatch">
                <property name="styleSheet">
                 <string notr="true">color: red;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="SignUp_button">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>DejaVu Serif</family>
                  <pointsize>13</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                  <stylestrategy>PreferAntialias</stylestrategy>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton:hover:pressed{
	background-color: rgb(27, 121, 209);
	color: #fff;
}

QPushButton:hover{
	background-color: #74A3EC;
	color: #fff;
}</string>
                </property>
                <property name="text">
                 <string>Sign up</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>8</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>149</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>49</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>149</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>49</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="CameraView">
       <property name="styleSheet">
        <string notr="true">QLabel#cameraLabel1{
	color: red;
}

QLabel#cameraLabel2{
	color: red;
}

QLabel#cameraLabel3{
	color: red;
}

QLabel#cameraLabel4{
	color: red;
}


QLabel#cam1{
	color: red;
	border: 1px solid black;
}
QLabel#cam2{
	color: red;
	border: 1px solid black;
}
QLabel#cam3{
	color: red;
	border: 1px solid black;
}
QLabel#cam4{
	color: red;
	border: 1px solid black;
}</string>
       </property>
       <widget class="QLabel" name="cam3">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>410</y>
          <width>67</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>No connection </string>
        </property>
       </widget>
       <widget class="QLabel" name="cam2">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>380</y>
          <width>69</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>No Connection </string>
        </property>
       </widget>
       <widget class="QLabel" name="cam4">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>440</y>
          <width>67</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>No connection </string>
        </property>
       </widget>
       <widget class="QLabel" name="cam1">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>16</y>
          <width>66</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>No Connection</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cameraSettings">
        <property name="geometry">
         <rect>
          <x>38</x>
          <y>1</y>
          <width>971</width>
          <height>761</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="CameraSettings">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: black;
}</string>
       </property>
       <widget class="QLineEdit" name="camera2LineEdit">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>180</y>
          <width>236</width>
          <height>17</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="camera2CheckBox">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>180</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Camera</string>
        </property>
       </widget>
       <widget class="QLabel" name="camera2Label">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>140</y>
          <width>307</width>
          <height>33</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera 2</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="camera4LineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>360</y>
          <width>236</width>
          <height>17</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="camera4CheckBox">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>360</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Camera</string>
        </property>
       </widget>
       <widget class="QLabel" name="camera4Label">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>340</y>
          <width>307</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera 4</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="camera3LineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>230</y>
          <width>236</width>
          <height>17</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="camera3CheckBox">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>230</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Camera</string>
        </property>
       </widget>
       <widget class="QLabel" name="camera3Label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera 3</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="camera1LineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>236</width>
          <height>17</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="camera1CheckBox">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>100</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Camera</string>
        </property>
       </widget>
       <widget class="QLabel" name="camera1Label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>41</width>
          <height>33</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera 1</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cameraName4LineEdit">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>450</y>
          <width>236</width>
          <height>17</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="cameraName3LineEdit">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>380</y>
          <width>236</width>
          <height>17</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="cameraName2LineEdit">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>310</y>
          <width>236</width>
          <height>17</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="cameraName1LineEdit">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>280</y>
          <width>236</width>
          <height>17</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="WhiteList1">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>150</y>
          <width>80</width>
          <height>15</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Manage white list</string>
        </property>
       </widget>
       <widget class="QPushButton" name="WhiteList2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>150</y>
          <width>80</width>
          <height>15</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Manage white list</string>
        </property>
       </widget>
       <widget class="QPushButton" name="WhiteList3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>150</y>
          <width>80</width>
          <height>15</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Manage white list</string>
        </property>
       </widget>
       <widget class="QPushButton" name="WhiteList4">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>150</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Manage white list</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="WhiteListTable">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>30</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="AddLine">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>310</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Add line</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SaveButton">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>310</y>
          <width>331</width>
          <height>151</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Serif</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="HomeButton">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>110</y>
          <width>251</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Serif</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1430</width>
     <height>15</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
